vagrant/forum/solution/forumdb_solved.py

```diff
@@ -0,0 +1,23 @@
+# Database code for the DB Forum, full solution!
+
+import psycopg2, bleach
+
+DBNAME = "forum"
+
+def get_posts():
```

Comment:
What are your thoughts about adding a refactor step after this one? Both of these functions have some common code that probably could be factored out (the connection, cursor, execution, and close.